python/pfs/pipe2d/gitChangelog.py

```diff
@@ -36,529 +25,187 @@
+"""Merged message regular expression (`re.Pattern`).
+The regular expression of the standard message generated when a ticket branch
+is merged to master.
 """
 
-GITHUB_API_TICKET_REGEX = r'^Subaru-PFS:tickets/([A-Z0-9]+-[0-9]+)$'
-"""JIRA ticket regular expression (`str`).
-The JIRA ticket name is stored in the GitHub API in the above
-format. This is used to parse the ticket number.
-"""
-
-TAG_REGEX = r"^([0-9]+)\.([0-9]+)(\.[0-9]+)?$"
-"""Git tag regular expression (`str`).
-The format of a tag. In the above case, M.m[.p] for releases.
+TAG_REGEX = re.compile(r"^w\.(\d{4})\.(\d{2})$")
+"""Git tag regular expression (`re.Pattern`).
+The format of a tag. In the above case, for weeklies eg w.2022.17 .
 """
 
 NOT_TAGGED = 'NOT-TAGGED'
 """Label for tickets not assigned to a tag (`str`).
 For grouping recently closed tickets that are not in a release, so not tagged.
 """
 
-DATE_TOL_MIN = 2
-"""The tolerance for comparing ticket timestamps against tag ranges [minutes]
-"""
-
-REPO_PRINCIPAL = 'pfs_pipe2d'
-"""Name of repo that provides tag-date information
-"""
-
-
-def _no_authentication(authfile):
-    """Handles situation where no GitHub API authentication
-    is to be provided.
 
-    Parameters
-    ----------
-    authfile : `str`
-        File containing authentication credentials
+def readAccessToken():
+    """Get access token for GitHub from file.
 
     Returns
     -------
-    auth : `tuple` (`set`, `set`)
-        tuple to enable HTTP authentication
+    authToken : `str`
+        Access token
     """
-    logger.debug('Using no authentication')
-    return None
+    logger.debug(f'Reading access token from {GITHUB_AUTH}..')
+    with open(GITHUB_AUTH) as fd:
+        return fd.readline().strip()
 
 
-def _external_authentication(authfile):
-    """Handles authentication being provided by an external file.
+def getCommitMessage(commit):
+    """Returns the corresponding message for the given commit.
 
     Parameters
     ----------
-    authfile : `str`
-        File containing authentication credentials
+    commit: `github.Commit.Commit`
+        Represents a git commit
 
     Returns
     -------
-    auth : `tuple` (`set`, `set`)
-        tuple to enable HTTP authentication
+    message: `str`
+        Commit message
     """
-    logger.debug('Authenticating using externally-supplied information')
-    if authfile is None:
-        logger.fatal('Need to supply a authentication file'
-                     'to the --authfile argument')
-        exit(1)
-    logger.debug(f'Authenticating using file {authfile}')
-    with open(authfile) as f:
-        parser = ConfigParser()
-        parser.read_file(f)
-        return (parser['github']['user'], parser['github']['token'])
+    return commit.raw_data['commit']['message']
 
 
-def _internal_authentication(authfile):
-    """Handles authentication being provided from the
-    Princeton internal network.
+def getTicketsForTags(repo, branch_name, sha_tag):
```

Comment:
This is a mix of `camelCase` and `under_scores`. You should choose one and stick to it.